src/cljs/main/broadfcui/page/workspace/notebooks/notebooks.cljs

```diff
@@ -258,25 +296,35 @@
                    (fn [cluster]
                      (if (or (= (:status cluster) "Running") (= (:status cluster) "Error"))
                        (links/create-internal
-                         {:data-test-id "x-button"
+                         {:data-test-id (str (:clusterName cluster) "-delete-button")
                           :id (:id props)
                           :style {:color (:text-light style/colors)
                                   :minHeight 30 :minWidth 30}
                           :onClick #(swap! state assoc :show-delete-dialog? true :cluster-to-delete (:clusterName cluster))}
                          (icons/render-icon {} :delete))))}
 
-                  {:header "Name" :initial-width 150
+                  {:header "Name" :initial-width 250
                    :as-text :clusterName :sort-by :clusterName :sort-initial :asc
                    :render
                    (fn [cluster]
-                     (if (= (:status cluster) "Running")
-                       (links/create-external {:href (leo-notebook-url cluster)} (:clusterName cluster))
-                       (:clusterName cluster)))}
-                  {:header "Status" :initial-width 100
-                   :column-data :status
-                   :render (fn [status]
-                             [:div {:style {:height table-style/table-icon-size}}
-                              (icon-for-cluster-status status) status])}
+                     (let [clusterName (:clusterName cluster)] (if (= (:status cluster) "Running")
+                       (links/create-external {:href (leo-notebook-url cluster)
+                                               :data-test-id (str clusterName "-link")} clusterName)
+                       clusterName)))}
+                  {:header "Status" :initial-width 150
+                   :as-text :status
+                   :render (fn [cluster]
+                             (let [clusterNameStatusId (str (:clusterName cluster) "-status")]
+                               [:div {:key (if clusters (str (gensym))) ;this makes the spinners sync
```

Comment:
@MattPutnam This seems to make the spinners sync :)

Reply:
Cool! `if` -> `when`